db/migrate/20191118183622_create_line_items.rb

```diff
@@ -1,19 +1,10 @@
 class CreateLineItems < ActiveRecord::Migration[6.0]
   def change
     create_table :line_items do |t|
-      t.references :product, null: false, foreign_key: true
-      t.belongs_to :cart, null: false, foreign_key: true
+      t.references :product, foreign_key: true
+      t.belongs_to :cart, foreign_key: true
```

Comment:
I'm not sure what you're doing here, but it is not a good idea to change a migration that has already been run. Rails runs each migration only once, so if you change the file after the migration has been run, the changes won't take affect. The best approach is to put the necessary changes in a new migration.